when.go

```diff
@@ -1,14 +1,15 @@
 package when
 
 import (
+	"github.com/olebedev/when/rules/nl"
```

Comment:
Can we move this to the section where all the other languages are being imported?

Reply:
Sure! Done :-)